test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -828,6 +829,132 @@ def init_proc_mon(self):
                                         frequency=freq)
                 self._process_monitoring = True
 
+    def revert_pbsconf(self):
+        """
+        Revert the following about the pbs.conf file:
```

Comment:
Since the function also restarts daemons when needed, It is better that we mention that in the docstring